packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -57,16 +60,23 @@ export class FocusTrapZone extends BaseComponent<IFocusTrapZoneProps, {}> implem
     const { className, ariaLabelledBy } = this.props;
     const divProps = getNativeProps(this.props, divProperties);
 
+    const bumperStyle: React.CSSProperties = {
+      pointerEvents: 'none'
+    };
```

Comment:
@dzearing I could use your eyes on this on potential side effects / issues with this style. I tried various other methods:
> display: none

This breaks onFocus callbacks
> display: contents

This breaks onFocus callbacks
> position: absolute, left: -9999px, top: -9999px

This was causing weird browser scrolling issues when changing focus